src/about/index.html

```diff
@@ -13,7 +13,7 @@ <h1 class="intro-section__headline">About Us</h1>
   <div class="section-content">
     <h1 class="about-page__heading">About DecodeMTL</h1>
     <p class="about-page__text">
-      We are currently running our courses out of the GriffinCamp a leading startup accelerator in downtown Montreal's innovation district.
+      We are currently running our courses out of the LaCommune a leading startup accelerator in downtown Montreal's innovation district.
```

Comment:
you changed the name, but you kept the same description. can you please modify the description to reflect what La Commune actually is?